adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/rest/v1/AdapterEndpoint.java

```diff
@@ -19,17 +19,17 @@
 @RequestMapping("/")
 public class AdapterEndpoint {
 
-    private final AdapterRepository adapterRepository;
+    private final AdapterFactory adapterFactory;
 
     @Autowired
-    public AdapterEndpoint(AdapterRepository adapterRepository) {
-        this.adapterRepository = adapterRepository;
+    public AdapterEndpoint(AdapterFactory adapterFactory) {
+        this.adapterFactory = adapterFactory;
     }
 
     @PostMapping("/dataImport")
-    public DataBlob.MetaData executeDataImport(@Valid @RequestBody AdapterConfig config) {
```

Comment:
Is there a reason you removed the "@valid@" annotation? 
As far as I know, the parameters are not being validated initially without the annotation (i.e. we need it to ensure there is a 400 response code if the request body is malformed).